scss/_offcanvas.scss

```diff
@@ -149,3 +149,9 @@
   padding: var(--#{$prefix}offcanvas-padding-y) var(--#{$prefix}offcanvas-padding-x);
   overflow-y: auto;
 }
+
+.offcanvas-dark {
```

Comment:
Do I need to split the PR to have the offcanvas-dark outside ?

Reply:
No need to split it in my opinion.